README.md

```diff
@@ -54,7 +54,14 @@ After the CloudFormation templates have been deployed, the [stack outputs](http:
 
 ### How to Deploy
 
-Stack is setup to launch stack in the us-west-2 (Oregon) region in your account:
+Stack is setup to launch stack in the us-west-2 (Oregon) region in your account.
+
+Launch the `master.yaml` from aws cli with command:
+```
+aws cloudformation create-stack --stack-name <your stack name> --template-body file://<absolute path to the local master.yaml> --capabilities CAPABILITY_NAMED_IAM --parameters  ParameterKey=KeyPairName,ParameterValue=<Name of your AWS account's key pair>
```

Comment:
What do we do to update CF with a new version of the `master.yaml` for an existing stack?  Do we just substitute `update-stack` parameter for the `create-stack` parameter here?